### PR TITLE
Databus -> data bus

### PR DESCRIPTION
**Pull request by kbaley**
Converts the term DataBus to Data Bus|Data bus|data bus depending on the context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ related:
 
 ## components.yaml
 
-"Components" is a general term used to describe a deployable set of functionality. Components exist in [components/components.yaml](components/components.yaml). Note that over time a component may have moved between nugets or split into new nugets. For example, the ABS DataBus or the Callbacks.
+"Components" is a general term used to describe a deployable set of functionality. Components exist in [components/components.yaml](components/components.yaml). Note that over time a component may have moved between nugets or split into new nugets. For example, the ASB Data Bus or the Callbacks.
```
Comment (SeanFeldman) [inline README.md]: There's no ASB DataBus, ASP.
Also, I think a component name should not have space.

```suggestion
"Components" is a general term used to describe a deployable set of functionality. Components exist in [components/components.yaml](components/components.yaml). Note that over time a component may have moved between nugets or split into new nugets. For example, the ASP Data Bus or the Callbacks.
```

File: nservicebus/messaging/databus/azure-blob-storage.md [hunk 2]
```diff
@@ -36,12 +36,12 @@ Review the [sample](/samples/azure/blob-storage-databus-cleanup-function/) to se
...
 
-Attachment blobs can be cleaned up using the [Blob Storage Lifecycle feature](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-lifecycle-management-concepts). This method allows configuring a single policy for all DataBus-related blobs. Those blobs can be either deleted or archived. The policy does not require custom code and is deployed directly to the storage account. This feature can only be used on GPv2 and Blob storage accounts, not on GPv1 accounts. 
+Attachment blobs can be cleaned up using the [Blob Storage Lifecycle feature](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-lifecycle-management-concepts). This method allows configuring a single policy for all data bus-related blobs. Those blobs can be either deleted or archived. The policy does not require custom code and is deployed directly to the storage account. This feature can only be used on GPv2 and Blob storage accounts, not on GPv1 accounts. 
 
 
 ## Configuration
 
-The following extension methods are available for changing the behavior of `AzureDataBus` defaults:
+The following extension methods are available for changing the behavior of `Azure Data Bus` defaults:
```
Comment (SeanFeldman) [inline nservicebus/messaging/databus/azure-blob-storage.md]: `AzureDataBus` is the returned type (see the [code](https://docs.particular.net/nservicebus/messaging/databus/azure-blob-storage#configuration)) and should not be changed.
```suggestion
The following extension methods are available for changing the behavior of `AzureDataBus` defaults:
```

**Comment by kbaley** [comment]
@adamralph For the most part, I use "data bus". The alternatives are used wherever I need to follow capitalization rules (e.g. start of a sentence, titles, etc.)

File: README.md [hunk 3]
```diff
@@ -261,7 +261,7 @@ related:
 
 ## components.yaml
 
-"Components" is a general term used to describe a deployable set of functionality. Components exist in [components/components.yaml](components/components.yaml). Note that over time a component may have moved between nugets or split into new nugets. For example, the ABS DataBus or the Callbacks.
+"Components" is a general term used to describe a deployable set of functionality. Components exist in [components/components.yaml](components/components.yaml). Note that over time a component may have moved between nugets or split into new nugets. For example, the ASP Data Bus or the Callbacks.
```
Comment (adamralph) [inline README.md]:
```suggestion
"Components" is a general term used to describe a deployable set of functionality. Components exist in [components/components.yaml](components/components.yaml). Note that over time a component may have moved between nugets or split into new nugets. For example, the ASB Data Bus or the Callbacks.
```

**Pull request merged**